src/main/kotlin/pt/up/fe/ni/website/backend/controller/EventController.kt

```diff
@@ -12,7 +12,7 @@ import pt.up.fe.ni.website.backend.service.EventService
 @RequestMapping("/events")
 class EventController(private val service: EventService) {
     @GetMapping
-    fun getAllEvents(): Collection<Event> = service.getAllEvents()
+    fun getAllEvents() = service.getAllEvents()
```

Comment:
I think this makes the functions more readable, since they're very simple. Do you agree?

Reply:
I think it is fine, maybe some will disagree tho

Reply:
There is a slight difference between having the type annotation and not having one. Without a type annotation, the return type of the function will be List<Event!> instead List<Event>. The exclamation mark means that the IDE doesn't know if the type is nullable. We should avoid these things to ensure as much null safety as possible, thus, writing out the type, at least in this case, is a good practice, in my opinion